File: Pot.java
import java.util.*;

class Pot{
  public static void main(String[] args) {
    Scanner kbd = new Scanner(System.in);
    int tot=0;
      int n=kbd.nextInt();
      for(int i=0;i<n;i++){
        int p = kbd.nextInt();
        int last = p%10;
        int remind = p/10;
        int calculate =  (int) Math. pow(remind,last);
        tot =tot+calculate;
      }   
      System.out.print(tot);
    
    kbd.close();
  }
}

File: pot.cpp

#include <iostream>
#include<math.h>
using namespace std;
int main() { 
      int tot=0,calculate=0;
      int n,p;
      cin>>n;
      for(int i=0;i<n;i++){
        cin>>p;
        int last = p%10;
        int remind = p/10;
        int calculate = pow(remind,last);
        tot =tot+calculate;
        
      } 
      cout<<tot<<endl;

}